yeti-platform/yeti
# Added virus total oneshot analytics

What?
Added virustotal free analytics api support for ip address and url's. I will continue to work, just wanted to push this an get feedback from the community.

How to test?

1. Start an investigation that contains an ip address or a hostname.
2. Click on any node and go to actions
3. You should see the virustotal action button, click on it (Make sure you add the virustotal api key in user settings and restart all the workers!)
4. You should see results

Is this over?
NO! Virus total still has files, hashes and more observables to query, this is only for network settings. I will complete the work when possible.
